### ISSUE TITLE
Genotyping for the inversions

### ISSUE THREAD

**Issue by shehongbing** [bioinformatics-centre/BayesTyper#39]
Hi,
     it's a powerful tool I used. So could the tool genotype for the inversions?

**Comment by jonassibbesen** [comment]
Yes, if you have a set of candidate inversions, BayesTyper should be able to genotype those.

**Comment by shehongbing** [comment]
Hi, 
    I have  another question. I want to genotype population on custom VCF file. 

As for the insertion or deletion, it is easy to generate the VCF. A deletion is as an example.
#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO
chr4  106924192       .       TGCCCAGGAAATTGAAG       T       .       .       .

it means a deletion occur in query.

However, what about the inversion.  For example, there is a inversion between ref and query.

ref, start - end (10 -20)  VS.  query, start - end (20 - 30).  Which is the correct format as follows:

(1) 
#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO
chr4  10       .       AAGGCCTTCC       AAGGCCTTCC       .       .       .

or (2)
#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO
chr4  10       .       AAGGCCTTCC       CCTTCCGGAA       .       .       .


Thx
